### ISSUE TITLE
AstroTelescope - LX200-compatible mount connected but not responding

### ISSUE THREAD

**Issue by illywhacker** [indigo-astronomy/CloudMakers-INDIGO-applications#18]
Hi,

I am trying to setup a Meade LX85 mount with AstroTelescope / INDIGO Server. The AudioStar handbox is connected to my MacBook via the Meade "505" cable and a serial-to-USB-C adapter with an FTDI chip. The FTDI driver is installed and the device is recognised.

Setting up INDIGO using the LX200 driver within AstroTelescope seems to work (all green), although it seems that no properties get updated from the mount. The mount doesn't respond to slewing or manual control and the reticle position is wrong.

I've attached a screenshot of the INDIGO control panel and a log.

Any help would be greatly appreciated!

Regards,

Nick

[astrotelescope.txt](https://github.com/indigo-astronomy/CloudMakers-INDIGO-applications/files/4568565/astrotelescope.txt)
<img width="1109" alt="Screenshot 2020-05-02 at 20 42 38" src="https://user-images.githubusercontent.com/20475789/80873180-080a0680-8cb7-11ea-9904-0a20688106c5.png">

**Comment by polakovic** [comment]
Hi Nick, log shows the mount didn't replied to any command. Are you really sure it is the correct port? Try to disconnect the mount and click refresh on serial ports property to see if there is a difference.

BTW, unless you have distributed setup NEVER use INDIGO Server. It is slower by many orders of magnitude compared to internal drivers in AstroTelescope.

Peter

**Comment by illywhacker** [comment]
Hi Peter,

Thank you so much for your swift reply. I did check that it's the correct
port by unplugging from the computer. When I unplug the mount-side nothing
happens, but I suppose that's normal since there's an active RS232-to-USB
converter.

However, I looked at the (brand new) Meade cable again and noticed that one
of the plugs isn't properly crimped... I'll try to fix that next week and
if that's the issue I already apologise for wasting your time!!

Regards,

Nick
